src/xdsl/dialects/scf.py

```diff
@@ -70,13 +70,13 @@ def verify_(self):
         for (idx, arg) in enumerate(self.arguments):
             if self.before_region.blocks[0].args[idx].typ != arg.typ:
                 raise Exception(
-                    f"Block arguments with wrong type, expected {arg.typ}, got {self.before_region.block[0].args[idx].typ}"
+                    f"Block arguments with wrong type, expected {arg.typ}, got {self.before_region.blocks[0].args[idx].typ}"
```

Comment:
This feels like an unrelated change.

Reply:
pyright report after_region does not have member .block
Upon looking at previous code with "before_region.blocks[0]", I believe what here means "after_region.blocks[0]"